render-utils.js

```diff
@@ -17,16 +17,31 @@ export function renderDogCard(dog) {
     return a;
 }
 
-export function renderDogDetail(dog) {
+export function renderDogDetail(dogData) {
     const div = document.createElement('div');
+    const nameEl = document.createElement('p');
     const img = document.createElement('img');
     const descriptionEl = document.createElement('p');
-    const nameEl = document.createElement('p');
+
     const ageEl = document.createElement('p');
     const breedEl = document.createElement('p');
     const ageAndBreedEl = document.createElement('div');
+    img.src = `../assets/${dogData.dog_breeds.name}.jpeg`;
 
     // examine the css classes already written and finish writing a detail page using the above elements
-
+    div.classList.add('dog-detail');
+    ageAndBreedEl.classList.add('age-and-breed');
+
+    nameEl.classList.add('name');
+    descriptionEl.classList.add('description');
+    img.classList.add('img');
+
+    descriptionEl.textContent = dogData.description;
+    nameEl.textContent = dogData.name;
+    ageEl.textContent = dogData.age;
+    breedEl.textContent = dogData.breed;
```

Comment:
your dogData object doesn't have a 'breed' key/value pair. the 'breed' is actually in the dog_breeds table under the 'name' column. if you target dogData.dog_breeds.name here you should get the 'breed' on the page.